tests/ensemble_evaluator/test_prefect_ensemble.py

```diff
@@ -589,18 +564,34 @@ def test_run_prefect_ensemble_exception(unused_tcp_port, coefficients):
         config.update(
             {
                 "config_path": os.getcwd(),
-                ids.REALIZATIONS: 2,
-                ids.EXECUTOR: "local",
-                "input_files": input_files(config, coefficients),
+                "realizations": 2,
+                "executor": "local",
+            }
+        )
+        inputs = {}
```

Comment:
This snippet of building the input and output of the configuration seems to be a reoccurring theme for the prefect ensemble tests. Worth to move into a function?

Reply:
Maybe, but we should not get too comfortable with this design, because it is problematic. ert3 owns the configuration of IO which prefect really wants to use. But it can't use it due to the then cyclical relation between the two.

The configuration needs to go out of ert3, but that's outside the scope of this PR (imo).